cppForSwig/BlockDataMap.h

```diff
@@ -48,11 +48,16 @@ struct BCTX
    const uint8_t* data_;
    const size_t size_;
 
+   BinaryData noWitData_;
+
    uint32_t version_;
    uint32_t lockTime_;
+   uint8_t  marker_;
```

Comment:
not useful here, stick to single bool flag, possibly even a const set at construction